includes/Traits/Publisher.php

```diff
@@ -40,6 +40,7 @@ trait Publisher {
 	 * Uses a fallback logo to always create valid AMP in FE.
 	 *
 	 * @since 1.0.0
+	 * @deprecated 1.1.0
 	 *
 	 * @return string
 	 */
```

Comment:
I tried to remove the `get_publisher_logo_placeholder` method. But I have to keep it as it is used in HTML class to remove placeholder image. With this logic, current stories with posters with the placeholder will load a 404 image See https://github.com/google/web-stories-wp/pull/4787/files#diff-9c6a425252305032ebe43f069d74d517R356

Reply:
Bummer :-(